source/plot/Axis.ooc

```diff
@@ -116,9 +116,9 @@ Axis: class {
 	getFirstTickValue: func -> Float {
 		result: Float
 		if ((this min < this tick && this min > 0.0f) || (this min > this tick && this min < 1.0f))
-			result = this min + (this tick - Float modulo(this min, this tick))
+			result = this min + (this tick - (this min modulo(this tick)))
```

Comment:
By the way, somebody should fix the names `minimum` and `maximum`, but that is hardly the responsibility if this PR.